sample_rules/exclusive_conditions.yaml

```diff
@@ -26,7 +26,7 @@
     # clear indicator it is indeed locked
     - condition: >
         lock_state == true &&
-        remote_key.command == 'lock'
+        remote_key == 'lock'
```

Comment:
These changes are not required anymore since dot notation is already supported.

Actually it'd be better to change the identifier to use full dot notation: **remote.key.command**